chore: implement new bundle structure

Davidmattei opened the pull request:
| Q              | A |
|----------------|---|
| Bug fix?       |  n |
| New feature?   | n  |
| BC breaks?     | n  |
| Deprecations?  | n  |
| Fixed tickets? | n  |
| Documentation? | n  |

Ready for review, do not check the 'mv ** commits (created with bash script see below)

About the new structure: https://symfony.com/doc/6.4/bundles/best_practices.html#directory-structure

After this pr you should do a new php bin/console assets:install --symlink in admin & web.


```bash
mv EMS/admin-ui-bundle/src/Resources/config EMS/admin-ui-bundle/config
git add -A && git commit -m "mv EMS/admin-ui-bundle/src/Resources/config EMS/admin-ui-bundle/config"
mv EMS/admin-ui-bundle/src/Resources/translations EMS/admin-ui-bundle/translations
git add -A && git add -A && git commit -m "mv EMS/admin-ui-bundle/src/Resources/translations EMS/admin-ui-bundle/translations"
mv EMS/admin-ui-bundle/src/Resources/views EMS/admin-ui-bundle/templates
git add -A && git commit -m "mv EMS/admin-ui-bundle/src/Resources/views EMS/admin-ui-bundle/templates"

mv EMS/client-helper-bundle/src/Resources/config EMS/client-helper-bundle/config
git add -A && git commit -m "mv EMS/client-helper-bundle/src/Resources/config EMS/client-helper-bundle/config"

.....
```

Davidmattei committed:
chore: fix linting no env=dev but APP_DEBUG=1

this way linting will work in the environment defined in the .env files

Davidmattei committed:
mv EMS/admin-ui-bundle/src/Resources/config EMS/admin-ui-bundle/config

Davidmattei committed:
mv EMS/admin-ui-bundle/src/Resources/views EMS/admin-ui-bundle/templates

Davidmattei committed:
mv EMS/admin-ui-bundle/src/Resources/translations EMS/admin-ui-bundle/translations

Davidmattei committed:
mv EMS/client-helper-bundle/src/Resources/config EMS/client-helper-bundle/config

Davidmattei committed:
mv EMS/client-helper-bundle/src/Resources/public EMS/client-helper-bundle/public

Davidmattei committed:
mv EMS/common-bundle/src/Resources/public EMS/common-bundle/public

Davidmattei committed:
mv EMS/common-bundle/src/Resources/config EMS/common-bundle/config

Davidmattei committed:
mv EMS/common-bundle/src/Resources/views EMS/common-bundle/templates

Davidmattei committed:
mv EMS/core-bundle/src/Resources/config EMS/core-bundle/config

Davidmattei committed:
mv EMS/core-bundle/src/Resources/DoctrineMigrations EMS/core-bundle/migrations

Davidmattei committed:
mv EMS/core-bundle/src/Resources/public EMS/core-bundle/public

Davidmattei committed:
mv EMS/core-bundle/src/Resources/views EMS/core-bundle/templates

Davidmattei committed:
mv EMS/form-bundle/src/Resources/config EMS/form-bundle/config

Davidmattei committed:
mv EMS/form-bundle/src/Resources/assets EMS/form-bundle/assets

Davidmattei committed:
mv EMS/core-bundle/src/Resources/translations EMS/core-bundle/translations

Davidmattei committed:
mv EMS/submission-bundle/src/Resources/views EMS/submission-bundle/templates

Davidmattei committed:
mv EMS/form-bundle/src/Resources/public EMS/form-bundle/public

Davidmattei committed:
mv EMS/submission-bundle/src/Resources/config EMS/submission-bundle/config

Davidmattei committed:
mv EMS/form-bundle/src/Resources/views EMS/form-bundle/templates

Davidmattei committed:
mv EMS/form-bundle/src/Resources/translations EMS/form-bundle/translations

Davidmattei committed:
fix: bundles should extend the AbstractBundle

Davidmattei committed:
fix: extensions should correctly load config

Davidmattei committed:
fix: find other resources usages

Davidmattei committed:
fix: routing use bundle config dir (no more resources)

Davidmattei committed:
fix: correctly load doctrine migrations

The coreBundle is also registering the namespace, no idea how this could work currently.

Davidmattei committed:
Merge branch '6.x' into chore/new-bundle-structure

theus77 reviewed (approved):
Sorry for the delay, it was not possible to review  on my smartphone 😂

Davidmattei committed:
Merge branch '6.x' into chore/new-bundle-structure

Davidmattei committed:
fix: no resources in asset builds

Tested asset build for admin-ui, core and formBundle

The form bundle should be migrated to vite
